### DISCUSSION TITLE
Joint: Updating frames after creation

### DISCUSSION THREAD

**Discussion by Freznosis** [NVIDIA-Omniverse/PhysX#125] [Q&A]
Is it possible to update frames of a joint after creation? If so, using Joint->SetLocalPose() after creation does nothing for me. If not, is there a recommended way of doing this? The only thing I can think of is to destroy the joint and recreate it every frame? Seems like that would be an odd work-around.

Thanks!

**Comment by preist-nvidia** [comment]
What are you trying to do? If your are trying to actuate actors relative to each other, you should use joint drives instead.

That being said, joint frame updates should work. Are you using GPU sim? Can you try CPU? Might be a bug that GPU sim is not correctly updated.

**Comment by Freznosis** [comment]
Was just trying to shift the pivot point of a spherical joint between two actors when a button is pressed. I think I was calculating relative offset between actors incorrectly, as it has been working fine. Thanks.
